From 61586c8cce93913b638173ecf0bc7e2ced65d9f1 Mon Sep 17 00:00:00 2001
From: Vasyl Gello <vasek.gello@gmail.com>
Date: Tue, 7 Apr 2020 03:18:22 +0000
Subject: [PATCH] Bump Security String to 2020-04-05

Implemented:
============
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2020-0070	A-148159613	RCE	Critical	8.0, 8.1, 9, 10
CVE-2020-0071	A-147310721	RCE	Critical	8.0, 8.1, 9, 10
CVE-2020-0072	A-147310271	RCE	Critical	8.0, 8.1, 9, 10
CVE-2020-0073	A-147309942	RCE	Critical	8.0, 8.1, 9, 10
CVE-2020-0081	A-144028297	EoP	High		8.0, 8.1, 9, 10

Not Implemented:
================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
None

Not Applicable (platform source):
=================================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2019-2056	A-140879284*	ID	High		10
CVE-2019-5018	A-140180629	EoP	High		8.0, 8.1, 9, 10
CVE-2019-8457	A-140182003	ID	High		8.0, 8.1, 9, 10
CVE-2019-9936	A-140181188	ID	Moderate	8.0, 8.1, 9, 10
CVE-2020-0078	A-144766455	EoP	High		9, 10
CVE-2020-0079	A-144506242	EoP	High		9, 10
CVE-2020-0080	A-144092031	EoP	High		10
CVE-2020-0082	A-140417434	EoP	High		10

Change-Id: Ica157a32f0da468c6cf1d5284eb80ecbfdb04a34
Signed-off-by: Vasyl Gello <vasek.gello@gmail.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index d30a9f0f0c..b3377d3670 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2020-03-05
+    PLATFORM_SECURITY_PATCH := 2020-04-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.17.1

